NDTensors/test/NDTensorsTestUtils/device_list.jl

```diff
@@ -18,13 +19,21 @@ if "metal" in ARGS || "all" in ARGS
 end
 if "cutensor" in ARGS || "all" in ARGS
   Pkg.add("cuTENSOR")
+  Pkg.add("CUDA")
   using CUDA, cuTENSOR
 end
 
+using JLArrays: JLArrays, jl
+
 function devices_list(test_args)
   devs = Vector{Function}(undef, 0)
   if isempty(test_args) || "base" in test_args
     push!(devs, NDTensors.cpu)
+    ## Skip jl on lower versions of Julia for now
+    ## all linear algebra is failing on Julia 1.6 with JLArrays
```

Comment:
I was wondering how you were handling testing JLArrays on older Julia versions where you were seeing issues, this seems like a reasonable solution. I guess JLArrays installs fine on Julia 1.6, but the tests fail?

Reply:
JLArrays does install fine and I was able to fix a symbol issue (JLArrays doesn't have `JLMatrix` defined but they do have `AnyJLMatrix` which is a larger union definition which includes some wrapper types). 
The issue with the JLArrays code in 1.6 is mainly that the `resize!` function is not included in the `JLArrays` version which is compatible with Julia 1.6. So i think its safer to just not test it on lower version especially because almost all GPU implementations don't support lower Julia versions